### PR TITLE
UpdatedConcordanceDoc

### PR DESCRIPTION
**Pull request by chandrans**
Simple fixes to capitalize VCF and changed command to reflect ./gatk-launch

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/Concordance.java [hunk 1]
```diff
@@ -27,27 +27,38 @@
...
  *
- * Optionally, the tool also produces vcfs of
- * 1) true positives and false negatives (i.e. all variants in the truth vcf)
- *      This mode is useful for calculating sensitivity
- * 2) true positives and false positives (i.e. all variants in the eval vcf)
- *      This mode is useful for obtaining a training data set for machine learning classifiers of artifacts.
+ * <p>Optionally, the tool also produces VCFs of the following:</p>
+ * <ul>
+ *     <li>True positives and false negatives (i.e. all variants in the truth VCF): useful for calculating sensitivity</li>
+ *     <li>True positives and false positives (i.e. all variants in the eval VCF): useful for obtaining a training data
```
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/Concordance.java]: Not sure I understand the point of these two -- what's different compared to the actual eval and truth vcfs we started with?

**Comment by vdauwera** [comment]
One question for either @takutosato or @davidbenjamin

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/Concordance.java (hunk 1 above)
Comment (davidbenjamin) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/Concordance.java]: @vdauwera These difference is that these have an additional `INFO` field tag for true positive, false negative, false positive, filtered true negative, and filtered false negative.  These tags turn the input vcfs into training data for classifiers.

**Comment by vdauwera** [comment]
Will merge once tests pass

**Pull request merged**